visualization/app/codeCharta/ui/metricChooser/metricChooser.component.scss

```
@@ -1,10 +1,28 @@
 $optionHeight: 48;
 
+cc-metric-value-hovered,
 area-metric-chooser-component,
 height-metric-chooser-component,
 color-metric-chooser-component,
 distribution-metric-chooser-component,
 edge-chooser-component {
+	.metric-value {
+		height: 25px;
+		float: right;
+		position: absolute;
+		right: 0;
+		margin-top: -9px;
+
+		.rounded-box {
+			margin-top: -2px;
+			margin-left: -2px;
+			background-color: #e6e6e6;
+			border-radius: 3px;
```

To keep the border radius consistent, id suggest using 2px like the other containers

```suggestion
			border-radius: 2px;
```